app/models/voucher.rb

```diff
@@ -1,15 +1,136 @@
 # frozen_string_literal: false
 
 class Voucher < ApplicationRecord
+  acts_as_paranoid
+
+  include CalculatedAdjustments
```

Comment:
Since we are not using a calculator any more and we are overriding the main method to create an adjustment, do we still need this module? Removing it would allow us to simplify the currently overriding method and avoid those Rubocop warnings straight away.

Reply:
Agreed, I originally used `CalculatedAdjustments` as a way to get started and to stay consistent with other adjustment, but we arrived at a solution where it doesn't really make sense anymore. It has now been removed